### PR TITLE
Get workflow name from `process.cwd`

### PR DESCRIPTION
**Pull request by johnisom**
This replaces the command line arguments for workflow/project name with a module that can be required as needed

### CODE REVIEW

File: bin/deploy.js [hunk 1]
```diff
@@ -11,7 +10,7 @@ const createStepFunction = require("../src/aws/createStepFunction");
 //   Specify files to retrieve by a workflow name
 const basenamesAndZipBuffers = getBasenamesAndZipBuffers("lambdas");
 const { lambdaRoleName, statesRoleName } = require("../src/config/roleNames");
-const stateMachineName = process.argv[2] || "example-workflow"; // TODO: perhaps throw an error?
+const stateMachineName = require("../src/util/workflowName");
```
Comment (T-monius) [inline bin/deploy.js:14]: I think line 13 can be removed. I can read in the `stateMachineName` in `src/aws/generateStateMachineParams.js`

File: src/util/workflowName.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+const cwd = process.cwd();
+const cwdBasename = cwd.replace(/\/.*\//, '')
+const cleanedCwdBasename = cwdBasename.replace(/\W/g, '-');
+
```
Comment (T-monius) [inline src/util/workflowName.js:4]: This is a good solution.

**Comment by johnisom** [comment]
This closes #35

File: bin/deploy.js [hunk 3]
```diff
@@ -10,7 +10,6 @@ const createStepFunction = require("../src/aws/createStepFunction");
 //   Specify files to retrieve by a workflow name
 const basenamesAndZipBuffers = getBasenamesAndZipBuffers("lambdas");
 const { lambdaRoleName, statesRoleName } = require("../src/config/roleNames");
-const stateMachineName = require("../src/util/workflowName");
```
Comment (johnisom) [inline bin/deploy.js]: It shows here that it is deleted

**Pull request merged**